bbs/menusupp.cpp

```diff
@@ -1033,3 +1040,23 @@ void SetDirNumber(const char *pszDirectoryKeys) {
     }
   }
 }
+
+void LogOffCmd() {
+    if (!a()->logoff_cmd.empty()) {
```

Comment:
TODO(rushfan): Make this logic into something reusable since we're doing it more than once and should probably support it in other places.